src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss

```diff
@@ -0,0 +1,49 @@
+$richie-organization-plugin-margin-bottom: $grid-gutter-width !default;
```

Comment:
Sorry! I meant every one of these variables exists to be overridden and needs `!default` 😉